src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+
+## New Notification APIs
+
+With the help of the Push and Notifications APIs, web applications have long been able to receive push messages and display notification banners. However, some parts were missing: Up until now, push messages had to be sent via the server; they could not be scheduled offline. Also, web applications installed to the system could not show badges on their icon. The Badging and Notification Triggers APIs enable both scenarios.
+
+### Badging API
+
+On several platforms, it's common for applications to present a badge on the application's icon indicating the amount of open actions: For instance, the badge could show the number of unread emails, notifications, or to-do items to complete. The [Badging API](https://web.dev/badging-api/) ([W3C Unofficial Draft](https://w3c.github.io/badging/)) allows installed web applications to show such a badge on its icon. By calling `navigator.setAppBadge()`, developers can set the badge. This method takes a number to be shown on the application's badge. The browser then takes care of displaying the closest possible representation on the user's device. If no number is specified, a generic badge will be shown (e.g., a white dot on macOS). Calling `navigator.clearAppBadge()` removes the badge again. Badging API is a great choice for email clients, social media apps, or messengers. The Twitter PWA makes use of the Badging API to show the number of unread notifications on the application's badge.
+
+{{ figure_markup(
+  image="badging_api.png",
+  alt="Percentage of page loads in Chrome using Badging API",
+  caption='Percentage of page loads in Chrome using Badging API. (Source: <a href="https://chromestatus.com/metrics/feature/timeline/popularity/2726">Chrome Platform Status metrics for Badge Set</a>, <a href="https://chromestatus.com/metrics/feature/timeline/popularity/2727">Chrome Platform Status metrics for Badge Clear</a>)',
+  description="Chart of Badging API usage, based on the percentage of page loads in Chrome using this feature. It compares the set and clear methods. The usage of both methods is growing over time, with the set method generally being called more often. In October 2020, there's a sudden growth for both methods, peaking at 0.025 percent of page loads for the set method and 0.016 percent for clear.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1145004925&format=interactive",
+  sheets_gid="1154751352"
+  )
+}}
+
+In April 2020, Google Chrome 81 shipped the new Badging API, followed by Microsoft Edge 84 in July. After Chrome shipped the API, the usage numbers shot up. In October 2020, on 0.025 percent of all page loads in Google Chrome, the `setAppBadge()` method is called. The `clearAppBadge()` method is less often called, during around 0.016 percent of page loads.
```

Comment:
Absolute page counts

Also, you mentioned that Twitter uses it. It could be insightful to query for all the URLs using it (or browse the sample URLs on the Chrome Status page itself) to find ones that are used on particularly large sites. Do you also see that this API has high usage in the % page views chart? That stat might be useful for comparison to make the point that few sites are using it but the ones that do are very large.

Reply:
Absolute page counts are not available (page loads), I'll have a look at the rest

Reply:
The stats use Chrome Platform Status data again (where we neither know the exact sites nor have absolute page counts available). HTTPArchive has only data available for the past three months. Both show a sudden growth in October. I don't see Twitter in the sample URLs though. The sample URLs seem to show various forums. It of course makes sense to present a badge if you decide to add the forum site to your homescreen. How should we proceed here?